### PR TITLE
Add simorgh naming description to readme

### PR DESCRIPTION
**Pull request by pjlee11**
_Add description of the reason for naming the application simorgh_

* ~[ ] Fixes https://github.com/bbc/simorgh/issues/NUMBER~
* ~[ ] Up-to-date with latest - `git pull --rebase origin latest`~
* ~[ ] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)~
* ~[ ] Tests added for new features~
* ~[ ] Tests pass - `npm test`~
* ~[ ] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)~

* ~[ ] Test engineer approval~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Simorgh
 
+Named simorgh after the Persian Mythological bird. The simorgh is the amalgam of many birds (and in some accounts other animals) into one. We consider this an apt metaphor for all articles of the BBC in one solution, a clear reference to the international nature of our teams but also to the articles themselves working for international users from the outset. It is also a unique name which is practical and, more superfically, the bird is very pretty.
```
Comment (sareh) [inline README.md]: If you could capitalise `Simorgh`, that would be great. Also, lowercae `mythological`.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Simorgh
 
+Named simorgh after the Persian Mythological bird. The simorgh is the amalgam of many birds (and in some accounts other animals) into one. We consider this an apt metaphor for all articles of the BBC in one solution, a clear reference to the international nature of our teams but also to the articles themselves working for international users from the outset. It is also a unique name which is practical and, more superfically, the bird is very pretty. 
+
+![](https://lh6.googleusercontent.com/OsrCnGr2OiqJEbpN7hkutKGqaYpS5M55MKNShymVcCJIk93rlxZLH5rvL3rj8JXphsf86OWh17itiION6WVaO1vU2GWpAOEBnpMSVihwPgUBP7TUirpEJrhkuWZ1xC6j9RK-l2Hg)
```
Comment (sareh) [inline README.md]: An alt attribute is needed here & we'd likely want to host this `simorgh` image in the `/docs` directory, correctly attributed to the creator of the image.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Simorgh
 
+Named Simorgh after the Persian mythological bird. The Simorgh is the amalgam of many birds (and in some accounts other animals) into one. We consider this an apt metaphor for all articles of the BBC in one solution, a clear reference to the international nature of our teams but also to the articles themselves working for international users from the outset. It is also a unique name which is practical and, more superfically, the bird is very pretty. 
+
+![](https://lh6.googleusercontent.com/OsrCnGr2OiqJEbpN7hkutKGqaYpS5M55MKNShymVcCJIk93rlxZLH5rvL3rj8JXphsf86OWh17itiION6WVaO1vU2GWpAOEBnpMSVihwPgUBP7TUirpEJrhkuWZ1xC6j9RK-l2Hg)
```
Comment (jamesbhobbs) [inline README.md]: Do we have permission to use this image? I couldn't find the copyright holder when I looked into doing this

Comment (pjlee11) [reply README.md]: I think we'd have to contact the gallery http://maryryangallery.com/artists/peter-sis/ to ask permission. There isn't anything about copyright on their website that I can see. We probably shouldn't use the image until we have confirmation.

**Review by jamesbhobbs — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Pull request merged**